src/command_modules/azure-cli-resource/tests/test_locks.py

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+
+class StorageAccountTests(ScenarioTest):
+    def test_list_locks(self):
+        locks_list = self.cmd('az lock list').get_output_in_json()
+        assert len(locks_list) == 0
+
+    def test_lock_create_list_delete(self):
+        for lock_type in ['ReadOnly', 'CanNotDelete']:
+            self.cmd('az lock create -n foo --lock-type {}'.format(lock_type))
+            locks_list = self.cmd('az lock list').get_output_in_json()
+            assert len(locks_list) > 0
```

Comment:
Suggest you check the `foo` shows up in the list like `'foo' in [l['name'] for l in locks_list]`